### PR TITLE
GROW-14 create buttons based on figma design

### PR DESCRIPTION
**Pull request by akowalska622**
I've created buttons based on Figma design. 

1. Buttons with text:
Zrzut ekranu 2021-08-20 o 18 56 33

They accept props:
- _outline_
- _alert_
- _children_ to render custom text

Primary styling is set by default.

2. Round buttons:
Zrzut ekranu 2021-08-20 o 18 57 13

They accept props:
- _outline_
- _alert_

Primary styling is set by default. Primary styling renders "+" sign, alert styling renders bin icon.

The path of UI folder and global CSS is absolute.

### CODE REVIEW

File: src/components/UI/Button.module.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
+@import 'css/constants/colors.module.css';
+@import 'css/constants/shadows.module.css';
+@import 'css/constants/fonts.module.css';
+
+.button {
+  color: white;
```
Comment (weronikaolejniczak) [inline src/components/UI/Button.module.css:6]: How about we write css instructions alphabetically?

File: src/components/UI/ButtonRound.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import classes from './ButtonRound.module.css';
+
+import { AiOutlineDelete, AiOutlinePlus } from 'react-icons/ai';
```
Comment (weronikaolejniczak) [inline src/components/UI/ButtonRound.js:3]: I'd prefer external imports to be at the top of the file, what do you think? :D

**Review by weronikaolejniczak — APPROVED** [review]
<3

**Comment by akowalska622** [comment]
I've changed CSS alphabetically and the order of imports

**Pull request merged**